src/main/ApplicationImpl.h

```diff
@@ -76,6 +76,7 @@ class ApplicationImpl : public Application
     virtual WorkScheduler& getWorkScheduler() override;
     virtual BanManager& getBanManager() override;
     virtual StatusManager& getStatusManager() override;
+    virtual SurveyDataManager& getSurveyDataManager() override;
```

Comment:
Could we make SurveyDataManager internal to SurveyManager, and let SurveyManager expose whatever is needed externally? Application is unfortunately a fairly huge class, and it already has a bit too many managers to keep track of. There should be minimal dependency on SurveyDataManager outside of survey, so better to encapsulate it as much as we can.

Reply:
Good idea. Done in 9ef7c8ce8f9c3dacca7a044585982f1e32084bf9